src/components/__internal__/abstract-custom-field/abstract-custom-field.test.jsx

```diff
@@ -27,6 +27,12 @@ describe('AbstractCustomField', () => {
       render(<AbstractCustomField {...sharedProps} ariaProps={ariaProps} />);
       expect(screen.getByLabelText('Test label')).toHaveAttribute('aria-controls', 'test-id-label');
     });
+
+    it('sets aria-describedby to link to a hint and aria-invalid when there are errors', () => {
+      render(<AbstractCustomField {...sharedProps} errorText={'there are errors.'} />);
+      expect(screen.getByLabelText('Test label')).toHaveAttribute('aria-describedby', 'test-idHint');
+      expect(screen.getByLabelText('Test label')).toHaveAttribute('aria-invalid', 'true');
```

Comment:
This is the better matcher: https://github.com/testing-library/jest-dom#tobeinvalid